### PR TITLE
better log and testing the tester

### PR DESCRIPTION
**Pull request by alan-stokes**

### CODE REVIEW

**Review by rowleya — APPROVED** [review]

**Review by rowleya — CHANGES_REQUESTED** [review]
You need a space I think

**Comment by rowleya** [comment]
Space and quotes required!

File: Jenkinsfile [hunk 1]
```diff
@@ -206,5 +206,5 @@ pipeline {
 
 def run_pytest(String tests, int timeout, String results, String threads) {
     sh 'echo "<testsuite tests="0"></testsuite>" > junit/' + results + '.xml'
-    sh 'py.test ' + tests + ' -rs -n ' + threads + ' --forked --show-progress --cov-config=.coveragerc --cov-branch --cov spynnaker8 --cov spynnaker --cov spinn_front_end_common --cov pacman --cov data_specification --cov spinnman --cov spinn_machine --cov spinn_storage_handlers --cov spalloc --cov spinn_utilities --junitxml junit/' + results + '.xml --cov-report xml:coverage.xml --cov-append --timeout ' + timeout
+    sh 'py.test ' + tests + ' -rs -n ' + threads + ' --forked --show-progress --cov-config=.coveragerc --cov-branch --cov spynnaker8 --cov spynnaker --cov spinn_front_end_common --cov pacman --cov data_specification --cov spinnman --cov spinn_machine --cov spinn_storage_handlers --cov spalloc --cov spinn_utilities --junitxml junit/' + results + '.xml --cov-report xml:coverage.xml --cov-append --timeout ' + timeout + '--log_level=INFO '
```
Comment (rowleya) [inline Jenkinsfile:209]: Space?

**Review by rowleya — APPROVED** [review]

**Comment by dkfellows** [comment]
This PR is testing for https://github.com/SpiNNakerManchester/SpiNNMan/pull/220

**Pull request merged**